pbjson-build/src/lib.rs

```diff
@@ -154,6 +155,14 @@ impl Builder {
         self
     }
 
+    /// Don't error out in the presence of unknown fields when deserializing, instead skip the
+    /// field only.
+    pub fn skip_unknown_fields(&mut self) -> &mut Self {
```

Comment:
I think for consistency with the terminology the proto documentation used for this feature, this should be called ignore_unknown_fields instead of skip_unknown_fields - https://developers.google.com/protocol-buffers/docs/proto3#json.

Reply:
Will update.